### ISSUE TITLE
How to train SOLOv2 from scratch

### ISSUE THREAD

**Issue by kojingying** [WXinlong/SOLO#111]
Hi, thank you for your great work!
I would like to replace all the ReLU activation function to mish, and need to train from scratch, may i know how to modify the config file for SOLOv2 like the learning rate and epochs needed and what else should i modify?
Thank you in advance!

**Comment by WXinlong** [comment]
@kojingying  I didn't try this. But to my knowledge, there are some papers focusing on training object detector from scratch. You may find some settings in those papers.
